platform/windows/crash_handler_windows.h

```diff
@@ -35,12 +35,15 @@
 #include <windows.h>
 
 // Crash handler exception only enabled with MSVC
-#if defined(DEBUG_ENABLED) && defined(_MSC_VER)
+#if defined(DEBUG_ENABLED)
```

Comment:
I noticed crash handler is disabled in release builds (also on macOS), but the Linux crash handler is available in release builds. This is needed to make https://github.com/godotengine/godot/pull/61906 effective in exported projects.

I suggest we always make the crash handler available in release builds, as release builds can be compiled with debug symbols enabled (or you can use a tool to reverse them if you have access to the debug symbols).